File: IssueTracker/IssueTracker/Source/IssueList/Model/Issue.swift
//
//  Issue.swift
//  IssueTracker
//
//  Created by 최예주 on 2022/06/14.
//

import Foundation

struct Issue: Codable {
    let title: String
    let description: String
    let milestone: Milestone?
    let tag: Tag?
    let state: IssueState?
}

File: app/src/main/java/com/example/it/issuetracker/domain/model/Issue.kt
package com.example.it.issuetracker.domain.model

import com.example.it.issuetracker.presentation.main.issue.list.Mode
import com.example.it.issuetracker.presentation.main.issue.list.Mode.DEFAULT
import java.io.Serializable

data class Issue(
    val id: Long,
    val title: String,
    val description: String,
    val label: List<Label>,
    var state: Boolean,
    val mileStone: String,
    var isChecked: Boolean = false,
    var viewType: Mode = DEFAULT,
    var isSwiped: Boolean = false,
) : Serializable